Dockerfile

```diff
@@ -29,5 +29,7 @@ COPY . /app
 # Expose the port that Gunicorn will listen on
 EXPOSE 5003
 
+RUN make load-mock-unit-data
```

Comment:
You might have to go back to the root directory before running this make command.

Reply:
I had to add a dependency to get the `make` command to work so it should be okay now 🤞 Thanks for the heads up. The files loaded from the dependent SDS schema branch will be from `main` branch, so will not be in a testable state at the moment until the dependent PR on that repo is merged